xmtp_mls/src/storage/encrypted_store/schema.rs

```diff
@@ -84,6 +84,13 @@ diesel::table! {
     }
 }
 
+diesel::table! {
```

Comment:
Is this still being used?

Reply:
Ah no, I missed it when I removed kvstore. I'll purge this.